test/DGmethods/compressible_Navier_Stokes/dycoms3d.jl

```diff
@@ -0,0 +1,829 @@
+        end
+    end
+
+    npoststates = 11
+    _int1, _int2, _betaout, _P, _u, _v, _w, _ρinv, _q_liq, _T, _θ = 1:npoststates
+    postnames = ("INT1", "INT2", "BETA", "P", "u", "v", "w", "rhoinv", "_q_liq", "T", "THETA")
+    postprocessarray = MPIStateArray(spacedisc; nstate=npoststates)
+
+    step = [0]
+    mkpath("vtk-dycoms")
+    cbvtk = GenericCallbacks.EveryXSimulationSteps(1000) do (init=false)
+        DGBalanceLawDiscretizations.dof_iteration!(postprocessarray, spacedisc,
+                                                   Q) do R, Q, QV, aux
+                                                       @inbounds let
+                                                          F_rad_out = radiation(aux)
+                                                           (R[_int1], R[_int2], R[_betaout], R[_P], R[_u], R[_v], R[_w], R[_ρinv], R[_q_liq], R[_T], R[_θ]) = (aux[_a_02z], aux[_a_z2inf], F_rad_out, preflux(Q, QV, aux)...)
+                                                       end
+                                                   end
+
+        outprefix = @sprintf("vtk-dycoms/cns_%dD_mpirank%04d_step%04d", dim,
+                             MPI.Comm_rank(mpicomm), step[1])
+        @debug "doing VTK output" outprefix
+        writevtk(outprefix, Q, spacedisc, statenames,
+                 postprocessarray, postnames)
+        #= 
+        pvtuprefix = @sprintf("vtk/cns_%dD_step%04d", dim, step[1])
+        prefixes = ntuple(i->
+        @sprintf("vtk/cns_%dD_mpirank%04d_step%04d",
+        dim, i-1, step[1]),
+        MPI.Comm_size(mpicomm))
+        writepvtu(pvtuprefix, prefixes, postnames)
+        =# 
+        step[1] += 1
+        nothing
+    end
+
+    # Initialise the integration computation. Kernels calculate this at every timestep?? 
+    integral_computation(spacedisc, Q, 0) 
+    solve!(Q, lsrk; timeend=timeend, callbacks=(cbinfo, cbvtk))
+
+
+#=
+    # Print some end of the simulation information
+    engf = norm(Q)
+    if integration_testing
+        Qe = MPIStateArray(spacedisc,
+                           (Q, x...) -> initialcondition!(Val(dim), Q,
+                                                          DFloat(timeend), x...))
+        engfe = norm(Qe)
+        errf = euclidean_distance(Q, Qe)
+        @info @sprintf """Finished
+        norm(Q)                 = %.16e
+        norm(Q) / norm(Q₀)      = %.16e
+        norm(Q) - norm(Q₀)      = %.16e
+        norm(Q - Qe)            = %.16e
+        norm(Q - Qe) / norm(Qe) = %.16e
+        """ engf engf/eng0 engf-eng0 errf errf / engfe
+    else
+        @info @sprintf """Finished
+        norm(Q)            = %.16e
+        norm(Q) / norm(Q₀) = %.16e
+        norm(Q) - norm(Q₀) = %.16e""" engf engf/eng0 engf-eng0
+    end
+integration_testing ? errf : (engf / eng0)
+=#
+
+end
+
+using Test
+let
+    MPI.Initialized() || MPI.Init()
+    Sys.iswindows() || (isinteractive() && MPI.finalize_atexit())
+    mpicomm = MPI.COMM_WORLD
+    if MPI.Comm_rank(mpicomm) == 0
```

Comment:
Change this to:
```suggestion
  ll = uppercase(get(ENV, "JULIA_LOG_LEVEL", "INFO"))
  loglevel = ll == "DEBUG" ? Logging.Debug :
  ll == "WARN"  ? Logging.Warn  :
  ll == "ERROR" ? Logging.Error : Logging.Info
  logger_stream = MPI.Comm_rank(mpicomm) == 0 ? stderr : devnull
  global_logger(ConsoleLogger(logger_stream, loglevel))
```
in order to have more MPI friendly logging.